dist/js/fuelux.js

```diff
@@ -2956,6 +2959,271 @@
+				this.$element.trigger( 'hidden.fu.superpicker' );
+			},
+
+			isExternalClick: function( e ) {
+				var el = this.$element.get( 0 );
+				var exceptions = this.options.externalClickExceptions || [];
+				var $originEl = $( e.target );
+				var i, l;
+
+				if ( e.target === el || $originEl.parents( '.superpicker:first' ).get( 0 ) === el ) {
+					return false;
+				} else {
+					for ( i = 0, l = exceptions.length; i < l; i++ ) {
+						if ( $originEl.is( exceptions[ i ] ) || $originEl.parents( exceptions[ i ] ).length > 0 ) {
+							return false;
+						}
+
+					}
+				}
+
+				return true;
+			},
+
+			show: function() {
+				var other;
+
+				if ( this.$element.hasClass( 'showing' ) ) {
+					return;
+				}
+
+				other = $( document ).find( '.superpicker.showing' );
+				if ( other.length > 0 ) {
+					if ( other.data( 'fu.superpicker' ) && other.data( 'fu.superpicker' ).options.explicit ) {
+						return;
+					}
+
+					other.superpicker( 'externalClickListener', {}, true );
+				}
+
+				this.previousValue = this.$field.val();
+
+				this.$element.addClass( 'showing' );
+
+				this.$popup.css( 'top', ( this.$field.outerHeight( true ) + 4 ) + 'px' );
+
+				this.$popup.css( 'height', ( this.options.height ) ? this.options.height : DEFAULT_HEIGHT + 'px' );
+				this.$body.css( 'height', ( ( this.options.height ) ? this.options.height : DEFAULT_HEIGHT ) - 73 + 'px' );
+				this.$popup.css( 'width', ( this.options.width ) ? this.options.height : DEFAULT_WIDTH + 'px' );
+
+				this.$element.trigger( 'shown.fu.superpicker', this.actualValue );
+				if ( this.actualValue !== null ) {
+					this.actualValue = null;
+				}
+
+				this.$field.attr( 'disabled', 'disabled' );
+
+				this.clickStamp = new Date().getTime() + ( Math.floor( Math.random() * 100 ) + 1 );
+				if ( !this.options.explicit ) {
+					$( document ).on( 'click.fu.superpicker.externalClick.' + this.clickStamp, $.proxy( this.externalClickListener, this ) );
+				}
+			},
+
+			setValue: function setValue( value ) {
+				this.$field.val( value );
+			},
+
+			getValue: function getValue() {
+				return this.$field.val();
+			}
+		};
+
+		// PLACARD PLUGIN DEFINITION
+
+		$.fn.superpicker = function( option ) {
+			var args = Array.prototype.slice.call( arguments, 1 );
+			var methodReturn;
+
+			var $set = this.each( function() {
+				var $this = $( this );
+				var data = $this.data( 'fu.superpicker' );
+				var options = typeof option === 'object' && option;
+
+				if ( !data ) {
+					$this.data( 'fu.superpicker', ( data = new Superpicker( this, options ) ) );
+				}
+
+				if ( typeof option === 'string' ) {
+					methodReturn = data[ option ].apply( data, args );
+				}
+			} );
+
+			return ( methodReturn === undefined ) ? $set : methodReturn;
+		};
+
+		$.fn.superpicker.defaults = {
+			onAccept: undefined,
+			onCancel: undefined,
+			externalClickAction: 'cancelled',
+			externalClickExceptions: [],
+			explicit: false,
```

Comment:
After reading the code I believe this means you have to explicitly close the pop-up instead of just clicking elsewhere, but I wonder if it could be more clearly named?